airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts

```diff
@@ -62,41 +66,125 @@ describe("Connection main actions", () => {
     deleteDestination(destName);
   });
 
+  it("Connection sync mode Incremental Append", () => {
+    const sourceName = appendRandomString("Test connection Postgres source cypress");
+    const destName = appendRandomString("Test connection Postgres destination cypress");
+
+    cy.intercept("/api/v1/web_backend/connections/update").as("updateConnection");
+
+    createTestConnection(sourceName, destName);
+
+    goToSourcePage();
+    openSourceDestinationFromGrid(sourceName);
+    openSourceDestinationFromGrid(destName);
+
+    goToReplicationTab();
+
+    searchStream("admins");
+    selectSyncMode("Incremental", "Append");
+    selectCursorField("email");
+
+    submitButtonClick();
+    confirmStreamConfigurationChangedPopup();
+
+    cy.wait(5000);
```

Comment:
This `cy.wait(5000)` is [not necessary](https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting): if you see timeout errors on the next line's `cy.wait("@updateConnection").then(/* test assertion */);` call, you can add the extra timeout to it directly by adding an addition options argument, like `cy.wait("@updateConnection", { timeout: 5000 }).then(/* test assertion */);`. You can safely make the timeout in this options argument very long: because it's part of the wait _for a specific network call_, cypress will stop waiting as soon as it sees the call, no matter how much time is left.